Typescript build error

In the latest version I get this build error:

node_modules/ical.js/dist/types/vcard_time.d.ts:51:5 - error TS2610: 'icaltype' is defined as an accessor in class 'Time', but is overridden here in 'VCardTime' as an instance property.

--- a/lib/ical/vcard_time.js
+++ b/lib/ical/vcard_time.js
@@ -83,7 +83,7 @@ class VCardTime extends Time {
    */
   constructor(data, zone, icaltype) {
     super(data, zone);
-    this.icaltype = icaltype || "date-and-or-time";
+    this._icaltype = icaltype || "date-and-or-time";
   }
 
   /**
@@ -99,7 +99,9 @@ class VCardTime extends Time {
    * @type {String}
    * @default "date-and-or-time"
    */
-  icaltype = "date-and-or-time";
+  get icaltype() {
+    return this._icaltype;
+  }
 
   /**
    * Returns a clone of the vcard date/time object.

